phase08/StudentGrades/StudentGrades/ProgramRunner.cs

```diff
@@ -0,0 +1,28 @@
+using DefaultNamespace;
+
+namespace project;
+
+public class ProgramRunner
+{
+    public void Run()
+    {
+        var fileReader = new FileReader();
+        var output = new OutputPrinter();
+        var dbController = DBController.Instance;
+        var dataParser = new JsonParser();
+
+        var studentsData = fileReader.ReadFileFromDisk(@"studentsData.json");
+        var scoresData = fileReader.ReadFileFromDisk(@"scoresData.json");
+
+        var students = dataParser.Parse<List<Student>>(studentsData);
+        var lessons = dataParser.Parse<List<StudentGrade>>(scoresData);
+
+        dbController.AddStudentsToDB(students);
+        dbController.AddGradesToDB(lessons);
+
+        dbController.InitAverageGrade();
+
```

Comment:
it might be better to put these three lines in one method called initDatabase to increase readability